.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -33,50 +33,19 @@ body:
 
 - type: dropdown
   attributes:
-    label: MSAL client type
+    label: Scenario
     description: "Are you using PublicClientApplication,  ConfidentialClientApplication or ManagedIdentityApplication?"
     multiple: true
     options: 
-      - "Public"
-      - "Confidential"
-      - "Managed identity"
+      - "PublicClient - desktop app"
```

Comment:
the plan was to use these values `Public`, `Confidential` and `Managed identity` and add auto tags

Reply:
Why can't we add tags based on these?

Reply:
I guess we could